From 26b707119bfef29e97783656bfd0df6bf0a774b1 Mon Sep 17 00:00:00 2001
From: Marcos Alano <mhalano@users.noreply.github.com>
Date: Fri, 18 Apr 2025 05:43:26 -0300
Subject: [PATCH] hwdb: add G-Mode key support (#37175)

Add G-Mode key, usually Fn+F9.

Closes #30824

(cherry picked from commit ae042183832bae0eecdb32877c568d5c9d67e646)

Resolves: RHEL-72701
---
 hwdb.d/60-keyboard.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 36bcdc9c3e..9e00d7cd83 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -383,6 +383,7 @@ evdev:name:gpio-keys:phys:gpio-keys/input0:ev:3:dmi:bvn*:bvr*:bd*:svncube:pni1-T
 ###########################################################
 
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnDell*:pn*:*
+ KEYBOARD_KEY_68=prog2                                  # G-Mode (Dell-specific)
  KEYBOARD_KEY_81=playpause                              # Play/Pause
  KEYBOARD_KEY_82=stopcd                                 # Stop
  KEYBOARD_KEY_83=previoussong                           # Previous song
